### PR TITLE
Indexer move to proper

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: backend/inspirehep/cli.py [hunk 1]
```diff
@@ -26,4 +27,4 @@
 cli.add_command(orcid)
 cli.add_command(mailing)
 cli.add_command(files)
-cli.add_command(sitemap)
```
Comment (monaawi) [inline backend/inspirehep/cli.py]: why did you remove the sitemap cli?

File: backend/inspirehep/records/indexer/tasks.py [hunk 2]
```diff
@@ -17,9 +17,9 @@
 )
 from sqlalchemy.orm.exc import NoResultFound, StaleDataError
 
+from inspirehep.indexer.base import InspireRecordIndexer
+from inspirehep.indexer.utils import get_record
 from inspirehep.records.api import AuthorsRecord, LiteratureRecord
-from inspirehep.records.indexer.base import InspireRecordIndexer
-from inspirehep.records.indexer.utils import get_record
```
Comment (monaawi) [inline backend/inspirehep/records/indexer/tasks.py]: maybe you should remove this since you moved it to indexer/tasks

File: backend/inspirehep/cli.py (hunk 1 above)
Comment (drjova) [reply backend/inspirehep/cli.py]: my bad

File: backend/inspirehep/records/indexer/tasks.py (hunk 2 above)
Comment (drjova) [reply backend/inspirehep/records/indexer/tasks.py]: wdym?

Comment (monaawi) [reply backend/inspirehep/records/indexer/tasks.py]: You added ```indexer/tasks```, why still keep ```records/indexer/tasks```?

Comment (drjova) [reply backend/inspirehep/records/indexer/tasks.py]: this is a leftover good catch
